msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -181,16 +177,31 @@ For more information, see [roles in a Teams meeting](https://support.microsoft.c
 
 After you design your app based on participant roles in a meeting, you can identify each user type for meetings and select what they can access.
 
+In public channel meetings, team owners can manage meeting app permissions through team settings.​ Channel meeting app permissions refer to the team app CRUD model (not the non-channel meeting app CRUD model). If a user can add an app to a team or channel, they can use it in the meeting.​
+
+While both team settings are enabled, meeting apps respect the existing permission model for non-channel meetings, where organizers and presenters can add apps but attendees cannot.​ In cases where users cannot add meeting apps, the entry points in pre-meeting and in-meeting experiences are removed.
```

Comment:
What are the 2 settings that "both team settings" refers to? There are 3 total settings highlighted in the screenshots below.

Reply:
please remove this as well.

Reply:
Updated as mentioned.